src/mango/src/mango_util.erl

```diff
@@ -85,14 +85,7 @@ open_doc(Db, DocId) ->
 
 
 open_doc(Db, DocId, Options) ->
-    case mango_util:defer(fabric, open_doc, [Db, DocId, Options]) of
```

Comment:
We shouldn't need defer anywhere now. Might make sense to delete it and assert that its not being used (unless of course I'm forgetting a situation we do still need it).

Reply:
I've added another commit that removes all mango_util:defer